tools/drug2cell/drug2cell.xml

```
@@ -0,0 +1,127 @@
+<tool id="drug2cell" name="drug2cell" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description>Collection of utility functions for gene group activity evaluation in scanpy</description>
+    <macros>
+        <token name="@TOOL_VERSION@">0.1.2</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="0.1.2">drug2cell</requirement>
+        <requirement type="package" version="1.10.4">scanpy</requirement>
+    </requirements>
+    <command>
+        <![CDATA[
+        ## Python script to execute drug2cell scoring and differential expression
+        python3 '$script'
+        ]]>
+    </command>
+    <configfiles>
+        <configfile name="script">
+import scanpy as sc
+import drug2cell as d2c
+
+# Load AnnData object
+adata = sc.read_h5ad("${adata}")
+
+# Combine ATC Level 1 and Level 2 into a single string
+categories = f"{level1}{level2}" 
```

In order to define / modify cheetah variabkls you need to 
```suggestion
#set categories = f"{level1}{level2}"
```

The error is hidden in the logs: https://github.com/galaxyproject/tools-iuc/actions/runs/12985253324/job/36209809943#step:9:818